README.md

```diff
@@ -894,7 +894,8 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 **[`^        back to top        ^`](#)**
 
 *See also [Wikis](#wikis)*
-
+ 
+- [Boostnote](http://dillinger.io/) - The note-taking app for programmers that focuses on markdown, snippets, and customizability. ([Source Code](https://github.com/BoostIO/Boostnote/)) `GPL-3.0` `JavaScript`
```

Comment:
the first link is still the entry to dillinger

Reply:
whoops !

Reply:
Just fixed it